api/EventType.json

```diff
@@ -0,0 +1,106 @@
+{
+  "api": {
+    "EventType": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/Events",
+        "support": {
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          },
+          "webview_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "click": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Events/click",
```

Comment:
The URL should be <code>https://developer.mozilla.org/docs/Web/Events/click</code>

Reply:
fixed.